gwpy/types/series.py

```diff
@@ -978,18 +979,20 @@ def crop(self, start=None, end=None, copy=False):
         if start == x0:
             start = None
         elif start is not None and xtype(start) < x0:
-            warn('%s.crop given start smaller than current start, '
-                 'crop will begin when the Series actually starts.'
-                 % type(self).__name__)
+            warn(
+                f"{type(self).__name__}.crop given start smaller than current "
+                "start, crop will begin when the Series actually starts.",
+            )
             start = None
 
         # pin late ends to time-series end
         if end == x1:
             end = None
         if end is not None and xtype(end) > x1:
-            warn('%s.crop given end larger than current end, '
-                 'crop will end when the Series actually ends.'
-                 % type(self).__name__)
+            warn(
+                f"{type(self).__name__}.crop given end larger than current "
+                "end, crop will begin when the Series actually ends.",
```

Comment:
'crop will begin when the series actually ends' -> 'crop will end when the series actually ends'?